.github/workflows/Benchmarks.yml

```diff
@@ -20,22 +30,14 @@ concurrency:
 jobs:
   benchmark:
     runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }}
     steps:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v2
         with:
           version: '1'
           arch: x64
-      - uses: actions/cache@v4
```

Comment:
Why don't we need these lines? I copied them from both the **Lux.jl** and the original example of the action

Reply:
I replace these lines with:
```
- uses: julia-actions/cache@v1
```

Reply:
Ok I see!